Morpher.API.AspNetSamples/Morpher.API.AspNetSamples.csproj

```diff
@@ -13,13 +13,14 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>AspNetSamples</RootNamespace>
     <AssemblyName>AspNetSamples</AssemblyName>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
```

Comment:
What about VS 7.0 and .NET 1.0? .Net 4.5 was released almost 6 years ago. Sometimes I can't install packages from nuget into .4.5 project, because no one want to target 6 years old framework. And you still care about people who use almost 8 years old framework. As far as i remember ws3.client target .net 4.0 but not a client profile version.